.pipelines/cni/load-test-templates/validate-state-template.yaml

```diff
@@ -12,9 +12,19 @@ steps:
       scriptType: "bash"
       addSpnToEnvironment: true
       inlineScript: |
-        make -C ./hack/aks set-kubeconf AZCLI=az CLUSTER=${{ parameters.clusterName }}-$(make revision)
+        if [ ${{ parameters.cnsManagedEndpoint }} == "true" ] && [ ${{ parameters.restartCase }} == "true"]

```

Comment:
`cnsManageEndpoint` is boolean right ? Are these getting logged in the pipeline ?